### DISCUSSION TITLE
Welcome to newquiz Discussions!

### DISCUSSION THREAD

**Discussion by joaomanaia** [joaomanaia/newquiz#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zoroya1** [comment]
Include app in Izzyondroid repo or fdroid ? Any plans ? It will be great as app will reach new users and more people can submit issues and contribute

**Reply by joaomanaia** [reply]
I already thought about including NewQuiz in F-Droid, the problem is that I use firebase and crashlytics, and F-Droid does not accept applications that have firebase and crashlytics by the inclusion policy. About Izzyondroid, I'll think about adding it to the repository.

**Reply by zoroya1** [reply]
Izzy repo is nice, it adds tag of `tracks or reports your activity` if app uses firebase, if user can turn on or off anonymous crash data or analytics, it will solve problem if someone wants to disable analytics
Looking forward to inclusion on izzyondroid repo.
Btw, thank you for great app, especially _logo quiz_ is so fun :)
Best wishes for app and for you <3
